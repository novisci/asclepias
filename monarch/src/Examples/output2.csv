concepts
"home"
"not home"
"home"
"home"
