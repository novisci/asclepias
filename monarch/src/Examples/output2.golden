"[[\"home\"],[\"not home\"],[\"home\"],[\"home\"]]"